### PR TITLE
Remove feature to count total test cases

### PR DESCRIPTION
**Pull request by namsonx**
- Duplicated
  - RobotFW core already has this feature

### CODE REVIEW

File: RobotFramework_Testsuites/Config/CConfig.py [hunk 1]
```diff
@@ -91,7 +91,6 @@ class CConfig():
     sProjectName      = None
     iSuiteCount       = 0
     iTestCount        = 0
-    iTotalTestcases   = 0
```
Comment (test-fullautomation) [inline RobotFramework_Testsuites/Config/CConfig.py:94]: Hi Son,
thank you four your investigation about this topic.
How does the robot code look like now to access the total number of test cases?
Thank you,
 Thomas

Comment (namsonx) [reply RobotFramework_Testsuites/Config/CConfig.py:94]: Hello Thomas,

The total testcaes is counted by ROBFW core already.
The core ROBFW count the number of test cases of every testsuite and sum all for total.

Thank you,
Son

Comment (test-fullautomation) [reply RobotFramework_Testsuites/Config/CConfig.py:94]: Hi Son,

O.K., but how can we access this.
The requirement is that we have a variable where we can calculate the progress of the test execution as part of our robot code.
Thank you,
 Thomas

Comment (namsonx) [reply RobotFramework_Testsuites/Config/CConfig.py:94]: Hello Thomas,

   In config object, we have iTestcount variable, it's auto increased in each test teardown => we can track test cases executed by using this variable.
   Currently, the number of test cases executed is printed out in our ROBFW-AIO log.

Thank you,
Son

Comment (test-fullautomation) [reply RobotFramework_Testsuites/Config/CConfig.py:94]: Hi Son,
very good. Please give me an example how to access iTestcount with robot code.
Thank you,
 Thomas

**Review by test-fullautomation — APPROVED** [review]
Hi Son,
looks good to me!
Thank you,
 Thomas

**Pull request merged**